sentry-core/src/main/java/io/sentry/core/SentryClient.java

```diff
@@ -35,6 +35,11 @@ public SentryClient(SentryOptions options, @Nullable AsyncConnection connection)
   public SentryId captureEvent(SentryEvent event, @Nullable Scope scope) {
     log(options.getLogger(), SentryLevel.DEBUG, "Capturing event: %s", event.getEventId());
 
+    // process events, should it be done before or after BeforeSecondCallback?
```

Comment:
@bruno-garcia

Reply:
Let's run event processors first. `BeforeSend` is the last thing we do before sending an event out to Sentry.